src/runtime_src/core/common/info_aie.cpp

```diff
@@ -299,14 +507,15 @@ populate_aie_shim(const xrt_core::device *device, const std::string& desc)
   try {
     boost::property_tree::ptree tile_array;
 
+    auto hw_gen = pt_shim.get<uint8_t>("hw_gen");
+
     // Populate the shim information such as dma, lock, error, events
     // for each tiles.
     for (const auto& as: pt_shim.get_child("aie_shim")) {
       const boost::property_tree::ptree& oshim = as.second;
       boost::property_tree::ptree ishim;
-      int col = oshim.get<uint32_t>("col");
-      int row = oshim.get<uint32_t>("row");
-
+      int col = oshim.get<int>("col");
```

Comment:
why do we need to change from uint32_t to int? 
Both of them works, but better not to change this.